README.rst

```diff
@@ -713,6 +713,12 @@ Choose from the list of available rules:
 
   Fix phpdoc inline tags, make inheritdoc always inline.
 
+  Configuration options:
+
+  - ``psr5`` (``bool``): whether the fixer should be PSR-5 compliant (uses
```

Comment:
PSR-5 specifically is not a code standard spec (it explicitly says this in the spec), so defining some code style options as being "psr5" perhaps doesn't make sense.